README.md

```diff
@@ -1,46 +1,47 @@
-# actions-npm-dependencies
+# The Guardian Package Linter
 
-Validate your NPM dependencies without installing Node (WIP)
+Lint your `package.json` in line with our recommendations for
+[dependencies](https://github.com/guardian/recommendations/blob/main/dependencies.md#javascript)
+and
+[packages](https://github.com/guardian/recommendations/blob/main/npm-packages.md).
 
-[Publint](https://publint.dev/@guardian/package-linter)
+**Please note** that in its current implementation, this linter will overwrite
```

Comment:
If you wanted fancy notes:
https://github.com/orgs/community/discussions/16925

Reply:
I worry about the accessibility implications: it’s not a quote!

Reply:
I'm with you, I did think it was a weird choice.
It actually renders as `<p>` though.

<img width="772" alt="Screenshot 2023-10-30 at 11 53 06" src="https://github.com/guardian/actions-npm-dependencies/assets/31692/dce4af3c-850f-4661-ac50-8701b8d22fbb">